imgui/cimgui.pxd

```diff
@@ -864,9 +864,9 @@ cdef extern from "imgui.h" namespace "ImGui":
             ImGuiInputTextFlags extra_flags
     ) except +  # Widgets: Sliders (tip: ctrl+click on a slider to input with keyboard.
     bool InputDouble(  # ✓
-            const char* label, double v,
+            const char* label, double* v,
```

Comment:
Hmm the signature in `imgui.h` is:
```c++
IMGUI_API bool          InputDouble(const char* label, double* v, double step = 0.0f, double step_fast = 0.0f, const char* format = "%.6f", ImGuiInputTextFlags extra_flags = 0);
```
And we're passing a pointer here so I think this should stay as `double* v`

Reply:
I don't understand.  My commit makes it pass the pointer to the double.

Reply:
🤦  nevermind

I'm just tired an misread the diff :D